txgh/lib/txgh/gitlab_repo.rb

```diff
@@ -0,0 +1,25 @@
+module Txgh
+  class GitlabRepo < Txgh::GithubRepo
+    def gitlab_config_branch
+      @gitlab_config_branch ||= process_all_branches? ? 'all' : Utils.absolute_branch(branch || 'master')
+    end
+
+    def gitlab_config_tag
+      @gitlab_config_tag ||= process_all_tags? ? 'all' : Utils.absolute_branch(tag) if tag
+    end
+
+    private
+
+    def should_process_branch?(candidate)
+      process_all_branches? ||
+        candidate.include?(gitlab_config_branch) ||
+        candidate.include?('L10N')
```

Comment:
This check is a super legacy thing left over from the very first version of txgh written by the folks at Strava. Should be able to remove it.

Reply:
Sure, will do